src/main/java/com/sams/demo/service/QuestionService.java

```diff
@@ -0,0 +1,9 @@
+package com.sams.demo.service;
+
+import com.sams.demo.data.Question;
+
+import java.util.List;
+
+public interface QuestionService {
```

Comment:
Let's use a bit different naming strategy for service interfaces - IQuestionService